src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +16,14 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        if (Objects.equals(email, users[0].getEmail())) {
+            return users[0];
+        }
+
+        if (Objects.equals(email, users[1].getEmail())) {
+            return users[1];
+        }
```

Comment:
I would say this solution is not flexible. For example, what if we have much more users, then just two?